test/base.js

```diff
@@ -132,4 +132,66 @@ describe('base config', () => {
         assert.equal(messages.length, 1);
         assert.equal(messages[0].ruleId, '@lwc/lwc/no-disallowed-lwc-imports');
     });
+
+    describe('should include no-restricted-imports', () => {
+        describe('prevents imports from @salesforce/lds', () => {
+            it('should prevent nested imports', async () => {
```

Comment:
nit: The two test cases share a lot of code. Suggestion: Create a function that accepts the source to lint and the messages to expect. Have the three tests call that function.